aldryn_forms/__init__.py

```diff
@@ -1,2 +1,2 @@
 # -*- coding: utf-8 -*-
-__version__ = '3.0.4'
+__version__ = '3.0.5'
```

Comment:
needs to be 4.0 according to the release notes

Reply:
@FinalAngel I will change it to 4.0 in a separate pull request as always we do.